frontend/packages/integration-tests-cypress/tests/storage/snapshot.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { testName, checkErrors } from '../../support';
+import {
+  SnapshotDetails,
+  snapshotClassDropdown,
+  dropdownFirstItem,
+  pvcDropdown,
+} from '../../views/storage/snapshot';
+import { listPage, status } from '../../views/list-page';
+import { PVC, testerDeployment, SnapshotClass } from '../../mocks/snapshot';
+import { detailsPage } from '../../views/details-page';
+import { submitButton } from '../../views/form';
+
+const snapshotName = `${PVC.metadata.name}-snapshot`;
+
+if (Cypress.env('BRIDGE_AWS')) {
+  describe('Snapshot Tests', () => {
+    before(() => {
+      cy.login();
+      cy.createProject(testName);
+      cy.exec(`echo '${JSON.stringify(PVC)}' | oc apply -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc apply -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(SnapshotClass)}' | oc apply -f -`);
+      cy.clickNavLink(['Storage', 'Persistent Volume Claims']);
+      listPage.filter.byName(PVC.metadata.name);
+      // Let the PVC get bound
+      cy.contains(status, 'Bound');
+    });
+
+    afterEach(() => {
+      checkErrors();
+    });
+
+    after(() => {
+      cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc delete -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(PVC)}' | oc delete -n ${testName} -f -`);
+      cy.exec(`echo '${JSON.stringify(SnapshotClass)}' | oc delete -f -`);
+      cy.deleteProject(testName);
+      cy.logout();
+    });
+
+    it('Creates Snapshot', () => {
+      cy.clickNavLink(['Volume Snapshots']);
+      listPage.clickCreateYAMLbutton();
+      cy.byTestID(pvcDropdown).click();
+      cy.get(dropdownFirstItem)
+        .first()
+        .click();
+      cy.byTestID(snapshotClassDropdown).click();
+      cy.get(dropdownFirstItem)
+        .first()
+        .click();
+      cy.get(submitButton).click();
+      cy.location('pathname').should(
+        'include',
+        `snapshot.storage.k8s.io~v1beta1~VolumeSnapshot/${PVC.metadata.name}-snapshot`,
+      );
+      detailsPage.titleShouldContain(PVC.metadata.name);
+      cy.contains(status, 'Ready');
+      return cy
```

Comment:
Shouldn't need this `return` anymore, recently disabled this eslint check.

Reply:
This rule related to "promise/catch-or-return" is not disabled. I will disable it and remove the return.

Reply:
Hmm...thought I did disable that one, but I see it isn't :-)  Thanks, please disable, and perhaps search for and fix any `return cy...`